cupy_setup_build.py

```diff
@@ -495,6 +510,16 @@ def preconfigure_modules(compiler, settings):
                     settings['library_dirs'].append(lib_path)
                     break
 
+        if module['name'] == 'cugraph':
+            cusparselt_path = os.environ.get('CUGRAPH_PATH', '')
+            for i in 'include', 'include/cugraph':
```

Comment:
Do we need to include `include/cugraph` as well?
I think just adding `include` dir to path list and using `#include <cugraph/algorithms.hpp>` from code looks better to avoid referencing the wrong header file.

Reply:
Thank you for pointing that out. I agree with that it is not clean to include `include/cugraph`. Actually, I tried the method you pointed out before, but it gave me the following error.

```
In file included from cupy_backends/cuda/libs/../../cupy_cugraph.h:12:0,
                 from cupy_backends/cuda/libs/cugraph.cpp:674:
/home/anaruse/.pyenv/versions/miniconda3-4.7.12/envs/libcugraph-nightly/include/cugraph/algorithms.hpp:17:10: fatal error: experimental/graph.hpp: No such file or directory
 #include <experimental/graph.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
error: command 'gcc' failed with exit status 1
```
This might be a problem with libcugraph, but I was able to solve it by adding `include/cugraph`, so I'm doing so now. Please let me know if you have any other good workarounds!

Reply:
Thank you for the explanation, so the current cugraph seems to expect `include/cugraph` to be on the path. Sounds reasonable to merge as is :+1: